man/fitNullModel.Rd

```diff
@@ -21,13 +22,15 @@
   association testing.
 
   \code{nullModelInvNorm} does an inverse normal transform of a previously fit null model.
+
+  \code{smallNullModel} returns a small version of the null model with no NxN matrices.
```

Comment:
A minor point, but should the names be either:
`smallNullModel` and `invNormNullModel`
or
`nullModelSmall` and `nullModelInvNorm`

Wondering if the symmetry is worthwhile.

Reply:
`nullModelInvNorm` is used in the analysis pipeline, so it would require updating the null model script there. I think it might be better if it's not symmetric - `fitNullModel` takes other inputs and returns a null model, while `nullModelInvNorm` and `nullModelSmall` take a null model as input (and return a modified null model).